packages/slice-machine/src/legacy/lib/builders/CustomTypeBuilder/UIDEditor/index.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActionButton,
+  DialogActions,
+  DialogCancelButton,
+  DialogContent,
+  DialogHeader,
+  FormInput,
+  Icon,
+  Text,
+} from "@prismicio/editor-ui";
+import {
+  Group,
+  NestableWidget,
+  UID,
+} from "@prismicio/types-internal/lib/customtypes";
+import { Formik } from "formik";
+import { type FC, useState } from "react";
+import type { AnyObjectSchema } from "yup";
+
+import { updateField } from "@/domain/customType";
+import { useCustomTypeState } from "@/features/customTypes/customTypesBuilder/CustomTypeProvider";
+import {
+  CustomTypes,
+  type TabField,
+  TabFieldsModel,
+} from "@/legacy/lib/models/common/CustomType";
+import { Widgets } from "@/legacy/lib/models/common/widgets";
+import { type Widget } from "@/legacy/lib/models/common/widgets/Widget";
+
+interface UIDEditorProps {
+  field: { key: string; value: TabField };
+}
+
+export const UIDEditor: FC<UIDEditorProps> = ({ field }) => {
```

Comment:
💡#nitpick

We are trying to merge our best practices with editor, you have here some things that don't follow, them, take a look here: https://github.com/prismicio/editor/blob/main/CONTRIBUTING.md#coding-guidelines

Note: we need to improve the discoverability of the best practices, but they are referenced in our Notion doc with a link to the contributing guide: https://www.notion.so/prismic/Slice-Machine-UI-s-Coding-Guidelines-d377c3e33faa44b3a5e0ad84a942616d